execute/exectypes/outcome.go

```diff
@@ -56,16 +54,16 @@ type Outcome struct {
 	// the state:
 	// * GetCommitReports: All pending commit reports which were observed.
 	// * GetMessages: All pending commit reports with messages.
-	// * Filter: Commit reports associated with the final execution report.
+	// * Filter: Commit reports associated with the final execution report, concatenated together.
 	CommitReports []CommitData `json:"commitReports"`
 
-	// Report is built from the oldest pending commit reports.
-	Report cciptypes.ExecutePluginReport `json:"report"`
+	// Reports are built from the oldest pending commit reports.
+	Reports []cciptypes.ExecutePluginReport `json:"reports"`
```

Comment:
The protobuf layer is managing the type migration, allowing the plugin layer to remove the legacy field.